media/js/base/core-datalayer-init.js

```diff
@@ -4,14 +4,31 @@
 
 // init core dataLayer object and push into dataLayer
 $(function() {
-    if (Mozilla && Mozilla.Analytics) {
-        var dataLayer = window.dataLayer = window.dataLayer || [];
+    var dataLayer = window.dataLayer = window.dataLayer || [];
```

Comment:
window.dataLayer is used all over the site already. If it was removed from the GTM snippet much breakage would already ensue, so I'm not sure adding this line everywhere is really worth it at this stage.